src/client/pages/components/chat/MessagePanel.jsx

```diff
@@ -3,7 +3,7 @@ import Message from './Message.jsx';
 
 export default function MessagePanel({ messages }) {
   return (
-    <div>
+    <div className="overflow-scroll max-h-full">
```

Comment:
using `overflow-scroll` shows a scroll bar at all times. `overflow-auto` instead would only show a scroll bar when needed.

Reply:
Oh ok nice! I couldn't see the scroll bar at all since it's black. Good to know.